src/studio/src/designer/frontend/dashboard/package.json

```diff
@@ -45,7 +45,7 @@
     "@types/classnames": "^2.3.1",
     "@types/enzyme": "^3.10.10",
     "@types/jest": "^27.0.3",
-    "@types/react": "17.0.34",
+    "@types/react": "17.0.37",
```

Comment:
@types/react should be reverted. See https://github.com/Altinn/altinn-studio/pull/7606